### ISSUE TITLE
Can't force Audio

### ISSUE THREAD

**Issue by phr34k0** [blissland/blissflixx#32]
Can't force Blissflix to play audio on headphone instead of HDMI

**Comment by blissland** [comment]
I don't think this is a Blissflixx issue. See:

https://www.raspberrypi.org/documentation/configuration/audio-config.md

**Comment by phr34k0** [comment]
Yes apparently it is, it doesn't send arguments to omxplayer with 

> -o local

command to force it to headphones instead of HDMI. Already tried multiple ways and recheck that by playing audio normally on browser and whatnot, it plays on headphones. Perhaps include it in the settings page for extra arguments to be inserted to omxplayer or peerflix upon play

**Comment by phr34k0** [comment]
OMX_CMD = "omxplayer --timeout 120 -I " in omxproc.py omxproc2.py is hardcoded :)

**Comment by blissland** [comment]
OK I see. Will look at making this configurable.

**Comment by gnshkl1** [comment]
Append option in omxproc.py omxproc2.py for audio in hdmi and analog device


-O both
